genie-web/src/main/java/com/netflix/genie/web/properties/ZookeeperLeadershipProperties.java

```diff
@@ -28,11 +28,17 @@
  * @author tgianos
  * @since 3.1.0
  */
-@ConfigurationProperties(prefix = "genie.zookeeper.leader")
+@ConfigurationProperties(prefix = ZookeeperLeadershipProperties.PROPERTY_PREFIX)
 @Getter
 @Setter
 @Validated
 public class ZookeeperLeadershipProperties {
+
+    /**
+     * The property prefix for this group.
+     */
+    public static final String PROPERTY_PREFIX = "genie.zookeeper.leader";
+
```

Comment:
Moving a constant to a different place in the same file doesn't make that big of a difference.
How about moving all of these prefixes to a class?
At least by looking at that it would be easier to come up with ways to stay consistent on naming future properties.

Reply:
It makes a difference in that I can reference them in other places now as none strings, see usage of `genie.grpc.server.enabled` in like 4 classes. Keeping it in the property class allows me (at least in my mental map) to scope it. If I have to go to a constants file now I have to name them all with huge long strings and if I delete the properties file at some point remember to go delete the dangling constant reference. This seems more self contained to me.